samples/python/tracker.py

```diff
@@ -3,8 +3,6 @@
 Tracker demo
 
 For usage download models by following links
-For GOTURN:
-    goturn.prototxt and goturn.caffemodel: https://github.com/opencv/opencv_extra/tree/c4219d5eb3105ed8e634278fad312a1a8d2c182d/testdata/tracking
```

Comment:
I think we should keep GOTURN at least in ONNX format

Reply:
We can use bytetracker, since converting goturn to onnx model may cost lots traffic.

Reply:
The GOTURN model will take 388 MB of traffic for each download if converted to onnx. If the user wants to use the tracking method, we can recommend they use Vit or dasimRPN, which have better performance. I create PR to remove it https://github.com/opencv/opencv/pull/25503